dwm1001/src/lib.rs

```diff
@@ -798,14 +803,14 @@ impl Led {
     pub fn enable(&mut self) {
         // https://github.com/braun-robotics/rust-dwm1001/issues/94
         #[allow(deprecated)]
-        self.0.set_low()
+        self.0.set_low().unwrap()
```

Comment:
I'd probably prefer `.ok()` to `.unwrap()` if you are discarding the errors anyway, or use `let _ = self.0.set_low();`. Unwarp can pull in a gross amount of formatting machinery.